src/lib/core/BUILD.gn

```diff
@@ -95,6 +95,8 @@ static_library("core") {
     "CHIPKeyIds.h",
     "CHIPTLV.h",
     "CHIPTLVDebug.cpp",
+    "CHIPTLVText.cpp",
```

Comment:
should we have a separate library for this?

Printing TLV seems to be useful for debugging/human output, however on embedded we probably do not want the overhead.

Reply:
There is no overhead if you don't include the symbols.